components/chi-siamo-card/ChiSiamoCard.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+    <div class="border border-gray-100 rounded-md p-8 m-8 bg-opacity-50 flex">
+        <!-- transform transition-all duration-300 ease-in scale-100 -->
```

Comment:
È rimasto un commento in più?

Reply:
Questa parte l'ha fatta @thedarkknight197 , lascio rispondere a lui